Documentation/network/kubernetes/local-redirect-policy.rst

```diff
@@ -531,8 +531,7 @@ When a local redirect policy is applied, cilium BPF datapath redirects traffic g
 However, for traffic originating from a node-local backend pod destined to the policy frontend, users may want to
 skip redirecting the traffic back to the node-local backend pod, and instead forward the traffic to the original frontend.
 This behavior can be enabled by setting the ``skipRedirectFromBackend`` flag to ``true`` in the local redirect policy spec.
-The configuration is supported with socket-based load-balancing and per packet based load-balancing, and requires ``SO_NETNS_COOKIE`` feature
-available in Linux kernel version >= 5.8.
+This configuration requires the ``SO_NETNS_COOKIE`` feature available in Linux kernel version >= 5.8.
```

Comment:
Were these doc changes left in on accident? There is no mention of them in the commit message or PR description.

Reply:
Actually, No. This was from a comment in https://github.com/cilium/cilium/pull/33721/commits/23ca9f758844f34569de57a8cdf0ebf2464aa092. 

 I split that into a separate commit and updated the PR description.

Reply:
Ah my apologies for missing that. Thanks for splitting it.